chrome-extension/content.js

```diff
@@ -189,14 +198,59 @@ async function getCategoryDropdown(userId) {
   return categoryDropdown;
 }
 
+/**
+ * Returns select HTML element of places search result options
+ * @param {String} text 
```

Comment:
`@param string? text` would indicate that the response can be null https://google.github.io/styleguide/jsguide.html#jsdoc-nullability

Reply:
Hey, Rica- sorry i didn't see this earlier! If you by any chance can still see this haha, I see on the style guide that a `?` before the type would mean the input parameter type can be null, which is not what we would mean right? Is the `?` being after the type what indicates that the response can be null?